opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -2309,17 +2316,20 @@ namespace Opm
         int it = 0;
         // relaxation factor
         double relaxation_factor = 1.;
-        const double min_relaxation_factor = 0.2;
+        const double min_relaxation_factor = 0.6;
         bool converged = false;
         int stagnate_count = 0;
+        bool relax_convergence = false;
         for (; it < max_iter_number; ++it, ++debug_cost_counter_) {
 
             assembleWellEqWithoutIteration(ebosSimulator, dt, inj_controls, prod_controls, well_state, deferred_logger);
 
             const BVectorWell dx_well = mswellhelpers::invDXDirect(duneD_, resWell_);
 
+            if (it > 40)
```

Comment:
Nitpick: braces please... (also line 2389).

Alternative (I like it, but not all might):
```
relax_convergence = it > 40;
```